eml/components/welcome/WelcomeSlider.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useRef } from "react";
+import { View } from "react-native";
+import Svg, { Path } from 'react-native-svg';
+import Sections from '../../constants/PreviewSections';
+import Slick from 'react-native-slick';
+import Text from '../general/Text';
+
+export default function WelcomeSlider() {
+  const slick = useRef(null);
+  const tailwindConfig = require('../../tailwind.config.js');
+  const projectColors = tailwindConfig.theme.colors;
+
+  return (
+    <Slick
```

Comment:
Is it not possible to use NativeWind here?

Reply:
nope 😢